dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/README.md

```diff
@@ -0,0 +1,211 @@
+# Jakarta Concurrency TCK Open Liberty Implementation
+
+This project is an implementation of the Jakarta Concurrency Technology Compatibility Kit (TCK) running on Open Liberty.
```

Comment:
Replace `This project is an implementation of the Jakarta Concurrency Technology Compatibility Kit (TCK) running on Open Liberty.`
with `This project runs the Jakarta Concurrency Technology Compatibility Kit (TCK) on Open Liberty.`